src/components/Guitar.tsx

```
@@ -0,0 +1,156 @@
+`
+
+const Fretboard = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-color: ${({ theme }) => theme.color.gray100};
+  border-radius: 5px;
+  overflow: hidden;
+  border: 1px solid ${({ theme }) => theme.color.gray200};
+`
+
+const String = styled.span<{ height: number; isActive?: boolean }>`
```

Biome (1.9.4)

[error] Do not shadow the global "String" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)